src/media/shortcodes/class-wpml-page-builders-media-shortcodes.php

```diff
@@ -44,6 +44,24 @@ public function translate( $content )  {
 		return $content;
 	}
 
+	/**
+	 * @param string $content
+	 *
+	 * @return bool
+	 */
+	public function has_media_shortcode( $content ) {
```

Comment:
@kagg-design, I suggest moving the code from https://github.com/OnTheGoSystems/wpml-page-builders/blob/4528acc9c636bd8db069dfda16c44fec5c5fd5a5/src/media/shortcodes/class-wpml-page-builders-media-shortcodes-update.php#L57 to the beginning of this method.

So basically, before to check if one shortcode is matching, we will check if there is any shortcode in the content.

Reply:
Done.